build/install-operator-sdk.sh

```diff
@@ -20,8 +21,8 @@ then
     fi
 fi
 
-PLATFORM="x86_64"
```

Comment:
can you explain these changes?

Reply:
This is a fix for #697 
Unrelated to this PR, but small so adding it here in another commit.